src/settingsmanager.cpp

```diff
@@ -186,5 +186,11 @@ void SettingsManager::initSettings()
     m_langList = m_settings.value("language", defaultLangVariant).toList();
 
     m_categoryList = m_settings.value("category", {}).toStringList();
+
+    /* Porting with older version: category pairs seperated with "|" 
+       are removed to avoid user confusion.
```

Comment:
Change the comment to

```c++
    /* Older versions used to save both the category identifier/code and its
     * name as shown to the user (separated by the | symbol). Now we store only
     * the category identifier from which the name is generated after loading.
     * In order to avoid user confusion we drop entries stored in the old format
     */
```

Reply:
Done